packages/fxa-auth-server/lib/routes/subscriptions.js

```diff
@@ -725,42 +775,40 @@ class DirectStripeRoutes {
     await this.updateCustomer(uid, email);
   }
 
-  async handleWebhookEvent(request) {
-    const event = this.stripeHelper.constructWebhookEvent(
-      request.payload,
-      request.headers['stripe-signature']
+  /**
+   * Handle `customer.source.expiring` Stripe wehbook events.
+   *
+   * @param {*} request
+   * @param {Event} event
+   */
+  async handleCustomerSourceExpiringEvent(request, event) {
+    const source = /** @type {Source} */ (event.data.object);
+    const { uid, email } = await this.sendSubscriptionPaymentExpiredEmail(
+      source
     );
+    await this.updateCustomer(uid, email);
```

Comment:
Why do we need to update the customer here?

Reply:
We might not need to - I've just been figuring a cache clear on any Stripe event is not a bad idea